integration

```diff
@@ -986,4 +986,4 @@
   "zeronounours/HA-custom-component-energy-meter",
   "zigul/HomeAssistant-CEZdistribuce",
   "ZsBT/hass-w1000-portal"
-]
\ No newline at end of file
+]
```

Comment:
This file should not be touched.
Adjust this.

Reply:
Would you like me to create a new pull request? Will that take another 3 months?

Reply:
Better to clean up the current one.